src/main/web/package.json

```diff
@@ -43,7 +43,7 @@
     "@angular/upgrade": "2.0.2",
     "@solgenomics/brapi-fieldmap": "git+ssh://git@github.com/IntegratedBreedingPlatform/brapi-fieldmap.git#semver:1.1.5",
     "@solgenomics/brapi-study-comparison": "git+ssh://git@github.com:IntegratedBreedingPlatform/BrAPI-Study-Comparison.git#semver:1.1.4",
-    "brapi-sync": "git+ssh://git@github.com/IntegratedBreedingPlatform/brapi-sync.git#semver:1.2.0",
+    "brapi-sync": "git+ssh://git@github.com/IntegratedBreedingPlatform/brapi-sync.git#IBP-5722-default-brapi-sync-base-url",
```

Comment:
:+1: 
before merge we do the [bump](https://ibplatform.atlassian.net/wiki/spaces/DEV/pages/1329627175/External+tools+and+BrAPPs#Working-with-the-code)

Reply:
brapi-sync version was bumped up to 1.3.0